Makefile

```diff
@@ -15,3 +15,18 @@ install:
 GOLANGCI_VERSION=v1.27.0
 lint:
 	@docker run --rm -t --user $$(id -u):$$(id -g) -v $$(go env GOCACHE):/cache/go -e GOCACHE=/cache/go -e GOLANGCI_LINT_CACHE=/cache/go -v $$(go env GOPATH)/pkg:/go/pkg -v ${PWD}:/app -w /app golangci/golangci-lint:${GOLANGCI_VERSION}-alpine golangci-lint run ./...
+
+
+release-dev:
+	env=dev make release
+
+release-prd:
+	env=prd make release
+
+PGP_FINGERPRINT=`secrethub read secrethub/logins/${env}/goreleaser/pgp/fingerprint.pgp`
```

Comment:
Do I understand this correctly that the `release-prd` will sign the binaries/archive, but the `release-dev` won't sign them? 

If so, is that desired behavior? Or do we also want to sign on `release-dev` (with a separate dev key)?

And final (obvious) question, won't goreleaser fail because `PGP_FINGERPRINT` isn't set?

Reply:
Both will sign the binaries, and both have their own separate `PGP_FINGERPRINT` and private key.

Reply:
I only tested `release-dev` but `release-prd` is basically the same commands with a different `PGP_FINGERPRINT` and a different private key.

Reply:
Sweet, well done! 👍